### ISSUE TITLE
 Forza Horizon 5: Premium Edition

### ISSUE THREAD

**Issue by sector24** [Fr33dan/GPSaveConverter#60]
Hello! I played the game through an online fix in the STEAM version, which includes DLC Hot Wheels and Rally Adventure. I decided to switch to the version from MS Store, installed the game, transferred the saves using the program. The saves were picked up, but for some reason the DLC does not work, more precisely, the game sees that they are there and bought, but does not open them, how to fix it?
![1](https://github.com/Fr33dan/GPSaveConverter/assets/80105112/a3a4647f-3525-4b97-9afb-581a789c5aed)
![2](https://github.com/Fr33dan/GPSaveConverter/assets/80105112/c254882b-3117-4382-88b2-4ab5cde8cdc2)
![3](https://github.com/Fr33dan/GPSaveConverter/assets/80105112/2eeaa657-5111-4b30-acfe-407a2a73b27d)

**Comment by sector24** [comment]
I added another screenshot, it shows that the DLC is not active. In the previous picture, you can see that the DLC was purchased, but it says that it's as if my game works through Gamepass and asks me to buy a separate DLC for it. I.e. it turns out that this is the problem - my saves were converted from STEAM to the Game pass version? is it possible to convert saves to the MS Store Premium edition version?
![4](https://github.com/Fr33dan/GPSaveConverter/assets/80105112/c4cdb34b-51f2-467c-abba-acfb0371aa9e)

**Comment by PeakAudioDevices** [comment]
The DLC's are activated by finding the correct spot on the game map. If you haven't started them before, they won't show up on the menu. Have you already played these? Also, I'm curious about getting my onlinefix save to work with a legit copy, my version of GPSaveConverter doesn't find any games and has an error that prevents it from even allowing me to look for them

**Comment by sector24** [comment]
> The DLC's are activated by finding the correct spot on the game map. If you haven't started them before, they won't show up on the menu. Have you already played these? Also, I'm curious about getting my onlinefix save to work with a legit copy, my version of GPSaveConverter doesn't find any games and has an error that prevents it from even allowing me to look for them

Yes, I've played DLC before. I solved the problem - it turned out to be in the MS Store account - because I bought an account for several users, then I had to delete one of the users (there should have been no more than 10 people), and the DLC worked. But another problem appeared - I was banned for the second time, it is unlikely that this is due to the transfer of saves, because earlier in the Steam version with an online fix, I was given warnings, and then banned for 5 days. And now the MS Store version has been blocked for 7 days. I don't know why they ban me, but many people complain. As for your question - I do not know why your saves are not being transferred. If you want to transfer the saves from the Steam version (online fix) to the MS Store version, then both games must be installed on the PC, then run the program from the administrator, select the game on the left, then specify the path to the Steam saves at the bottom, select 5 files similar to MS Store, and click on the arrow, and everything is ready.
